eclass/kernel-install.eclass

```diff
@@ -38,13 +45,93 @@ esac
 
 inherit dist-kernel-utils mount-boot toolchain-funcs
 
+LICENSE="GPL-2"
+_INITRD_LICENSE=()
+if [[ ${KERNEL_IUSE_GENERIC_UKI} ]]; then
+	# Keep in sync with the dracut modules included in our prebuilt images!
+	#
+	# Apache-2.0 for dev-libs/openssl, app-crypt/argon2
```

Comment:
Ok, idea two: let's make a list of packages and licenses, and use it both to construct the license here and the `BDEPEND` in the other eclass. Then you wouldn't need comments at all, and we could also look into autogenerating the list from ebuilds.

Reply:
Something like this?:
```
	INITRD_PACKAGES=(
		"dev-libs/openssl Apache-2.0"
		"app-crypt/argon2 || ( Apache-2.0 CC0-1.0 )"
		"dev-libs/libbsd BEER-WARE BSD BSD-2 BSD-4 ISC MIT"
                 ...
	)
       export INITRD_PACKAGES
       BDEPEND="generic-uki? ( ${INITRD_PACKAGES%% *} )"
       LICENSE="geneirc-uki? ( ${INITRD_PACKAGES#* } )
```

Reply:
For example, except you don't need to `export` it. Or associative array, whichever works better.